config-default.yml

```diff
@@ -206,6 +206,7 @@ guild:
         staff_announcements:    &STAFF_ANNOUNCEMENTS    464033278631084042
 
         # Voice Channels
+        afk_voice:                          756327105389920306
```

Comment:
[`Guild.afk_channel`](https://discordpy.readthedocs.io/en/latest/api.html#discord.Guild.afk_channel) provides the AFK channel of the guild already.